src/Bicep.Core.UnitTests/Utils/RegistryHelper.cs

```diff
@@ -153,7 +153,7 @@ public static async Task PublishProviderToRegistryAsync(IDependencyHelper servic
             throw new InvalidOperationException($"Failed to get reference '{errorBuilder(DiagnosticBuilder.ForDocumentStart()).Message}'.");
         }
 
-        await dispatcher.PublishProvider(targetReference, new(tgzData));
+        await dispatcher.PublishProvider(targetReference, new(tgzData, false, []));
```

Comment:
Yeah, my personal style is to avoid named args by default. I will sometimes use them if I feel like they add clarity, but IMO they often get in the way of code readability without adding any useful context.

Reply:
IMHO I'd argue that having a naked "false" or "true" argument is much less readable.  When possible I prefer to redesign the API (e.g. use an enum), but otherwise I like using named args for this case.